connection.go

```diff
@@ -135,6 +135,11 @@ func (mc *mysqlConn) Prepare(query string) (driver.Stmt, error) {
 }
 
 func (mc *mysqlConn) interpolateParams(query string, args []driver.Value) (string, error) {
+	// Number of ? should be same to len(args)
```

Comment:
Yes. Current implementation doesn't care queries like `SELECT 'foo?bar', ?`.
When we add support it, this check should go below.